components/grid-col/src/index.js

```diff
@@ -14,8 +14,7 @@ const colValues = {
 };
 
 const widthFromProps = spacing.withWidth({ noDefault: true });
-const desktopWidthFromProps =
-  spacing.withWidth({ mediaQuery: MEDIA_QUERIES.DESKTOP, noDefault: true });
+const desktopWidthFromProps = spacing.withWidth({ mediaQuery: MEDIA_QUERIES.DESKTOP, noDefault: true });
```

Comment:
surely that is more than 120 characters

Reply:
nope - it's 105

Reply:
ok, prettier will deal with line length for us